machine/agema/agema_ag7648c/machine.make

```diff
@@ -0,0 +1,42 @@
+# AGEMA AG7648C
+# CPU Module: Intel Atom Rangeley (C2338)
+
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = bcm
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# AGEMA/Delta IANA number
+VENDOR_ID = 2254
+
+I2CTOOLS_ENABLE = yes
+I2CTOOLS_SYSEEPROM = no
+
+# This platform requires the PXE_EFI64 installer
+PXE_EFI64_ENABLE = no
+
+CONSOLE_SPEED = 115200
+CONSOLE_DEV = 0
+CONSOLE_FLAG = 0
+
+EXTRA_CMDLINE_LINUX = i2c-ismt.bus_speed=100 acpi=off dhcp=eth0
```

Comment:
there are two ethernet ports on the cpu board, eth0 is for OOB, eth1 is for internal debug.
 "dhcp=eth0" is to make sure the onie use the OOB port only, otherwise onie will use eth0 and eth1.